.github/workflows/root-ci.yml

```diff
@@ -366,7 +366,7 @@ jobs:
             overrides: ["CMAKE_CXX_STANDARD=23"]
           - image: alma8
           - image: alma9
-            overrides: ["CMAKE_BUILD_TYPE=Debug"]
+            overrides: ["CMAKE_BUILD_TYPE=Debug", "CMAKE_CXX_FLAGS=-Wsuggest-override"]
```

Comment:
If the CI would be red without solving the `mathcore` problem, we could also consider not changing the CI configuration yet and merged the rest of this PR. Then we can add this flag to the CI once the math problem is resolved (just in case there is no other way to silence the warnings in `mathcore`).